### DISCUSSION TITLE
(Node-Red-contrib-tasmota) MQTT Nodes no longer report or forward device status in flows, instead showing offline, BUT still control devices

### DISCUSSION THREAD

**Discussion by digitydogs** [arendst/Tasmota#18715] [Q&A]
I know this issue is basically with another project, but it is directly connected to this one so I am reaching out here as well. 

Trying to figure out the cause of this issue, and I cannot pinpoint anything in the tasmota changelogs that may be related to this, so I am posting here in hopes of getting some input on what may have changed that is affecting this that i just overlooked.

I recently updated all of my devices (40+) to tasmota 12.5 after being on 9.2/9.5 for quite some time. I am still able to control all of my lights and other devices via the nodes, but now they no longer report anything to do with the state/status.  

I can see the information arriving to the tele,stat, and cmnd topics, so my only thought is that maybe something in the stat topics data format changed, but I cannot tell what. 

any ideas?

Tasmota 12.5.0.3
Node Red 3.0.2
nore-red-contrib-tasmota 0.9.12

**Comment by barbudor** [comment]
The problem is somewhere else

Tasmota 12.5.0.3
Node Red 3.0.2
nore-red-contrib-tasmota 0.9.12

![image](https://github.com/arendst/Tasmota/assets/2996042/443f5c89-7896-4911-8928-0ba8de99d160)

I personnaly don't see the need for that Tasmota integration and do everything with standard MQTT nodes.

**Reply by digitydogs** [reply]
This kind of attitude and response is the reason so many people are dropping tasmota for esphome and so few  feel comfortable recommending tasmota these days.  I reached out here for assistance because the ONLY thing that changed was the tasmota version and I was trying to find out WHAT changed between versions as I can't seem to find mention of what is or could be causing my issue after 2 days of researching it.

**Reply by barbudor** [reply]
What attitude?
I'm a happy user of Tasmota
I kindly tried to reproduced your problem with very little information you provided and I can't.
It's working fine on my side as you see my device is reported online 

So unless you can provide more details I strongly suggest that you report the problem to the designer of  the node which would be the only one to explain what is the problem in his node.
